lib/core/src/server/manager/manager-webpack.config.js

```diff
@@ -55,7 +55,7 @@ export default ({
       refs
         ? new VirtualModulePlugin({
             [path.resolve(path.join(configDir, `generated-refs.js`))]: refsTemplate.replace(
-              `'{{refs}}'`,
+              /'{{refs}}'/g,
```

Comment:
I'm unsure why this is necessary, `'{{refs}}'` only occurs once in the template?

Reply:
true, the `g` is not needed